lib/puppet-strings.rb

```diff
@@ -53,14 +53,18 @@ def self.generate(search_patterns = DEFAULT_SEARCH_PATTERNS, options = {})
     YARD::CLI::Yardoc.run(*args)
 
     # If outputting JSON, render the output
-    if options[:json]
+    if options[:json] && !options[:describe]
       render_json(file)
     end
 
     # If outputting Markdown, render the output
     if options[:markdown]
       render_markdown(file)
     end
+
+    if options[:describe]
```

Comment:
If we want to have `--describe` as a generate option, we'll need to register it in the face https://github.com/puppetlabs/puppet-strings/blob/master/lib/puppet/face/strings.rb#L7

on further examination this is just something that the action passes to generate, i guess.

Reply:
Yes, I am just using it to carve out a space in the generate options, since I expect most of the describe options to not be applicable to the other modes.  Probably a little awkward style wise.